terraform-beyond-layer0/example-2/output.tf

```diff
@@ -0,0 +1,8 @@
+#
+# ./example2/output.tf
+#
+
+# Note the syntax for referencing resources using the count property
+output "guestbook_urls" {
```

Comment:
this should be a `list` output, not a string. For reference, see: https://github.com/terraform-community-modules/tf_aws_vpc/blob/master/outputs.tf#L25